docs/static/cross-plugin-concepts.asciidoc

```diff
@@ -10,8 +10,15 @@ List-type URI parameters will automatically expand strings that contain multiple
 whitespace-delimited URIs into separate entries. This behaviour enables the expansion
 of an arbitrary list of URIs from a single Environment- or Keystore-variable.
 
-Examples of plugins and options that support this functionality:
+The following plugins and options support this functionality:
 
 * <<plugins-inputs-elasticsearch-hosts,Elasticsearch input plugin - `hosts`>>
 * <<plugins-outputs-elasticsearch-hosts,Elasticsearch output plugin - `hosts`>>
 * <<plugins-filters-elasticsearch-hosts,Elasticsearch filter plugin - `hosts`>>
+
+For example, users use this functionality to define an environment variable with
+multiple whitespace-delimited URIs to use it for the options above, e.g.:
+
+```
+ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
+```
```

Comment:
```suggestion
You can use this functionality to define an environment variable with
multiple whitespace-delimited URIs and use it for the options above.

**Example**
```
ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
```
```

Reply:
Did you overlook this suggestion or choose not to take it?  If you'd like to discuss, I can explain the thinking behind the suggestion. 
https://github.com/elastic/logstash/pull/12672#discussion_r577219716